docs/en/final-task.md

```diff
@@ -2,15 +2,15 @@
 
 ## FAQ
 ### Question: Who makes assessment of the course project?
-Answer: Our mentor does verification of intermediate steps. The final assessment makes another randomly chosen mentor.
+Answer: Our mentor does verification at the intermediate steps. The final assessment makes another randomly chosen mentor.
 
-### Question: Does course project assessment being made as a review of regular task?
+### Question: Does the course project assessment being made as a review of regular task?
 Answer: Yes, in a dialogue format. Mentor who makes review leaves comments and can ask to fix something or explain.
```

Comment:
Mentor who reviews the project leaves comments and can ask to fix something or to clarify something.